features/steps/steps.py

```diff
@@ -0,0 +1,56 @@
+"""
+Order Steps
+Steps file for orders.feature
+"""
+from os import getenv
+import json
+import requests
+from behave import *
+from compare import expect, ensure
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions
+
+WAIT_SECONDS = 120
+BASE_URL = getenv('BASE_URL', 'http://localhost:5000/')
+
+@given('the following orders')
+def step_impl(context):
```

Comment:
I dont think these steps are being run since the corresponding scenario in the features file is commented out